tests/cli/test_integration_datasets.py

```diff
@@ -24,49 +24,43 @@
 from renku import cli
 
 
+@pytest.mark.parametrize(
+    'doi', [(
+        '10.5281/zenodo.597964', 'y', 'pyndl_naive_discriminat_v064',
+        'K.Sering,M.Weitz,D.Künstle,L.Schneider'
+    ), ('10.7910/DVN/S8MSVF', 'y', 'hydrogen_mapping_laws_a_1', 'M.Trevor')]
+)
 @pytest.mark.integration
-def test_dataset_import_real_doi(runner, project):
+def test_dataset_import_real_doi(runner, project, doi):
     """Test dataset import for existing DOI."""
     result = runner.invoke(
-        cli.cli, ['dataset', 'import', '10.5281/zenodo.597964'], input='y'
+        cli.cli, ['dataset', 'import', doi[0]], input=doi[1]
     )
     assert 0 == result.exit_code
     assert 'OK' in result.output
 
     result = runner.invoke(cli.cli, ['dataset'])
+    print(result.output)
     assert 0 == result.exit_code
-    assert 'pyndl_naive_discriminat_v064' in result.output
-    assert 'K.Sering,M.Weitz,D.Künstle,L.Schneider' in result.output
+    assert doi[2] in result.output
+    assert doi[3] in result.output
 
 
 @pytest.mark.parametrize(
-    'doi', [
-        ('10.5281/zenodo.597964', 'y'),
-        ('10.5281/zenodo.3236928', 'n'),
-        ('10.5281/zenodo.2671633', 'n'),
-        ('10.5281/zenodo.3237420', 'n'),
-        ('10.5281/zenodo.3236928', 'n'),
-        ('10.5281/zenodo.3188334', 'y'),
-        ('10.5281/zenodo.3236928', 'n'),
-        ('10.5281/zenodo.2669459', 'n'),
-        ('10.5281/zenodo.2371189', 'n'),
-        ('10.5281/zenodo.2651343', 'n'),
-        ('10.5281/zenodo.1467859', 'n'),
-        ('10.5281/zenodo.3240078', 'n'),
-        ('10.5281/zenodo.3240053', 'n'),
-        ('10.5281/zenodo.3240010', 'n'),
-        ('10.5281/zenodo.3240012', 'n'),
-        ('10.5281/zenodo.3240006', 'n'),
-        ('10.5281/zenodo.3239996', 'n'),
-        ('10.5281/zenodo.3239256', 'n'),
-        ('10.5281/zenodo.3237813', 'n'),
-        ('10.5281/zenodo.3239988', 'y'),
-        ('10.5281/zenodo.3239986', 'n'),
-        ('10.5281/zenodo.3239984', 'n'),
-        ('10.5281/zenodo.3239982', 'n'),
-        ('10.5281/zenodo.3239980', 'n'),
-        ('10.5281/zenodo.3188334', 'y'),
-    ]
+    'doi', [('10.5281/zenodo.597964', 'y'), ('10.5281/zenodo.3236928', 'n'),
```

Comment:
Did yapf do this? I've wanted to keep it 1 per line on purpose (easier to insert new ones without touching old)

Reply:
I think it was yapf, I kept it at 1 per line. I'll see if it does it again